contributors/README.template.md

```diff
@@ -0,0 +1,17 @@
+# NucleoidMC
+
+Hello and welcome to NucleoidMC!
+
+Nucleoid is an effort to build a fully open-source minigames server on modern versions of Minecraft.
+
+You can join us with Minecraft: Java Edition at: `nucleoid.xyz`.
+
+If you want to get involved, you can find us:
+- [On Discord](https://nucleoid.xyz/discord)
+- [On GitHub](https://github.com/NucleoidMC/)
+
+## Contributors
+
+Nucleoid wouldn't be where it is today without all of our ✨ amazing ✨ contributors:
```

Comment:
Maybe the ✨ sparkles ✨ make it seem a bit not genuine? 😄

Reply:
Hmm, I can see what you mean :+1: